ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/ip-security.j2

```diff
@@ -0,0 +1,61 @@
+{#
+ Copyright (c) 2023 Arista Networks, Inc.
+ Use of this source code is governed by the Apache License 2.0
+ that can be found in the LICENSE file.
+#}
+{# doc - ip security #}
+{% if ip_security is arista.avd.defined %}
+
+### IP Security
+{%     if ip_security.ike_policie is arista.avd.defined %}
```

Comment:
```suggestion
{%     if ip_security.ike_policies is arista.avd.defined %}
```

Reply:
Already fixed in [8053791](https://github.com/aristanetworks/ansible-avd/pull/3194/commits/80537919832da472d38eaff495688e138a5e88e5) - think this comment was made on a previous version of the PR